app/src/main/java/com/onegini/mobile/exampleapp/view/activity/MobileAuthenticationFingerprintActivity.java

```diff
@@ -36,7 +36,7 @@ protected void setupUi() {
 
   @Override
   protected void setCancelButtonVisibility() {
```

Comment:
This method doesn't `override` anymore and can be private. The same rule applies for other activities changed in this MR